build.zig

```diff
@@ -27,16 +27,28 @@ pub fn build(b: *Builder) !void {
     });
     b.default_step.dependOn(&fmt_step.step);
 
+    const test_enum = enum {
+        UNIT,
+        ALL_RUNTIME,
+        NORMAL,
+        PANIC,
+    };
+
+    comptime var available_tests: []const u8 = "";
+    inline for (@typeInfo(test_enum).Enum.fields) |field| {
+        available_tests = available_tests ++ field.name ++ ", ";
```

Comment:
You could add a description of what each test mode does so the user knows what to choose. You could do so by making a function in `test_enum` that returns the description and adding it to the string (perhaps within parentheses) here.

Reply:
A function sounds good